requirements.txt

```diff
@@ -1,3 +1,7 @@
+# Developers: Please do not modify this auto-generated file. If
+# you want to add, remove, upgrade, or downgrade libraries,
+# please change the `requirements.in` file, and then follow
+# the instructions there to regenerate this file.
```

Comment:
Is this change intended?

Reply:
this is fine. after I added checksums in the requirements file, so many other developers were pushing this change into their PRs, so pushing it to develop from here will fix it